VideoLocker/res/values/strings.xml

```diff
@@ -344,7 +344,7 @@
     <!-- Button text to complete an action -->
     <string name="done_text">Done</string>
     <!-- Label indicating unread actions. Argument is a count -->
-    <string name="unread_text">%s Unread</string>
+    <string name="unread_text">{unread_num} Unread</string>
```

Comment:
maybe use ``unread_count``. We should also update the comment so it says something like ``{unread_count} is the number of unread messages``. Also, I think this string is buggy. It looks like the code passes an integer not a string. This will really need to be pluralized, I guess I missed it when I was updating those.